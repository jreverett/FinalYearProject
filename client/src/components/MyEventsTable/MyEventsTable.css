#event-table-container {
  height: 94.5%;
  background-image: linear-gradient(#fff, #d4f7f0);
  border-radius: 3px;
  box-shadow: 2px 2px 5px rgba(1, 1, 1, 0.5);
  overflow: scroll;
}
