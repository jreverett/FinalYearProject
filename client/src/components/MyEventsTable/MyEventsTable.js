import React, { Component } from 'react';
import { MyEventsRow } from '..';
import '../../common.css';
import './MyEventsTable.css';

class MyEvents extends Component {
  render() {
    var ownedEvents = this.props.loggedInUser.ownedEvents;
    var ownedEventItems = [];

    if (ownedEvents) {
      ownedEventItems = ownedEvents.map((event, index) => {
        return (
          <MyEventsRow
            key={index}
            userID={this.props.loggedInUser._id}
            eventID={event}
          />
        );
      });
    }
    return (
      <div className="table-container">
        <div>{ownedEventItems}</div>
      </div>
    );
  }
}

export default MyEvents;
