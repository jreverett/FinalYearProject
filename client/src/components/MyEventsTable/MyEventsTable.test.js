import React from 'react';
import renderer from 'react-test-renderer';
import MyEventsTable from './MyEventsTable';

describe('<MyEventsTable />', () => {
  it('matches the snapshot', () => {
    const testUser = { ownedEvents: [] };
    const tree = renderer
      .create(<MyEventsTable loggedInUser={testUser} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
