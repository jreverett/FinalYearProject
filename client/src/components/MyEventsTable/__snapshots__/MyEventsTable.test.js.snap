// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyEventsTable /> matches the snapshot 1`] = `
<div
  className="table-container"
>
  <div />
</div>
`;
